src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -428,7 +428,7 @@
   height: var(--pf-c-dropdown__menu-item-icon--Height);
   margin-right: var(--pf-c-dropdown__menu-item-icon--MarginRight);
 
-  > * {
+  > img {
```

Comment:
This can be any sort of image/icon/thing. I think the main use cases are `<img>` and `<svg>` but it could be whatever.